### ISSUE TITLE
Hangs at "ethereum loading" window, "btc-swap contract not found" in console

### ISSUE THREAD

**Issue by michbil** [etherex/etherex#12]
When trying to open etherex in web browser, when connected to local geth node, "Ethereum Loading" windows appears and hags there forever.

In browser console there's error: 
```
btc-swap contract not found
```

**Comment by temsa** [comment]
Same problem here

**Comment by caktux** [comment]
Contracts haven't been deployed on the network yet, they were on Olympic and will be on the official testnet once it lands in the next geth release.

**Comment by caktux** [comment]
EtherEx is now on the Morden testnet, and the UI was updated to work with the contracts deployed on it.

**Comment by alexfisher** [comment]
Having this same issue.  Seems to hang around 0-2 blocks left (even though geth is downloading them).  I installed ntp to ensure my time is syncing, but it seems like either my geth client is just enough behind that the EtherEx UI doesn't consider it caught up, or there's a bug?

**Comment by alexfisher** [comment]
FWIW, when I browse directly to localhost:8545 I get:

{"id":-1,"jsonrpc":"2.0","error":{"code":-32600,"message":"Could not decode request"}}

And, looking at the EtherEx UI, it says last block was 3 seconds ago, waiting for 0 blocks, and in the background I can see an error listed on the sidebar that says:

"Error loading accounts: Error: could not decode, invalid type on field value: not a string"

Come to think of it, I just initialized testnet and very likely don't have an account created on local geth testnet instance.  hmmm... I'll go check...

**Comment by alexfisher** [comment]
Yep, that was it.  Created an etherbase account and it loads up like a charm now.  Probably not the first or last person to make this mistake, at least!

**Comment by tomachinz** [comment]
<img width="1411" alt="eth loading" src="https://cloud.githubusercontent.com/assets/1026291/24857079/2019c028-1e3b-11e7-87f7-b2e87e6ee458.png">

**Comment by tomachinz** [comment]
So I would need to launch the contracts on my private net myself yes? How does one do that? I can see three contracts and a YAML file if only I knew how to deploy those into my private net....
